### ISSUE TITLE
no cookie-cache specified for newsblur cryptic message is displayed 

### ISSUE THREAD

**Issue by bs76** [newsboat/newsboat#245]
**Newsboat version (copy from `newsboat -v`)**:
```
newsboat 2.10.2 - https://newsboat.org/
Copyright (C) 2006-2015 Andreas Krennmair
Copyright (C) 2015-2017 Alexander Batischev
Copyright (C) 2006-2017 Newsbeuter contributors
Copyright (C) 2017 Newsboat contributors

newsboat is free software and licensed under the MIT License.
Type `newsboat -vv' for more information.

newsboat 2.10.2
System: Linux 4.13.0-45-generic (x86_64)
Compiler: g++ 7.2.0
ncurses: ncurses 6.1.20180127 (compiled with 6.0)
libcurl: libcurl/7.58.0 GnuTLS/3.5.18 zlib/1.2.11 libidn2/2.0.4 libpsl/0.19.1 (+libidn2/2.0.4) nghttp2/1.30.0 librtmp/2.3 (compiled with 7.57.0)
SQLite: 3.22.0 (compiled with 3.21.0)
libxml2: compiled with 2.9.4
```

Steps to reproduce the issue:

1. config file contains:
```
urls-source "newsblur"
newsblur-login "<whatever>"
newsblur-passwordfile "~/.newsboat/newsblur_passwd"
```
2. config file does *not* contain `cookie-cache`

3. Starting newboat:

```
Starting newsboat 2.10.2...
Loading configuration...done.
Opening cache...done.
 is inaccessible and can't be created
```

**Other info you think is relevant**:

`cookies` from https://github.com/newsboat/newsboat/blob/master/src/controller.cpp#L289
is not set and results in cryptic message here
https://github.com/newsboat/newsboat/blob/master/src/controller.cpp#L292

**Comment by Minoru** [comment]
Thanks for the report!

But I'm a bit confused: in step 1 you show a config *with* `cookie-cache` but in step 2 you say there's no such setting in the config file. Is that just a mistake, or am I completely misunderstanding the issue?

I [just added](https://github.com/newsboat/newsboat/commit/6b9f899ef4c353fe0e2078e437d31c6213e802a1) another message that will be displayed if the setting is not set or is set to an empty value. This change will be released in 2.13 in September. Does that resolve the issue?

**Comment by bs76** [comment]
You're right, I messed it up, the line with `cookie-cache` should not be there. I did change the description.

And btw. thanks, you do awesome work!
